Scripts/Services/City Loyalty System/CityLoyaltyEntry.cs

```diff
@@ -132,9 +132,7 @@ public void CheckTradeDeal()
         public override void Serialize(GenericWriter writer)
         {
             base.Serialize(writer);
-            writer.Write(2);
-
-            writer.Write(ArtisanBodPoints);
+            writer.Write(1);
```

Comment:
Should it not still be Write (2) ?

Reply:
nope i changed it, but handled the bod points a different way. I forgot to take that out.